snapshot/bazel/constants.go

```diff
@@ -19,11 +19,15 @@ const (
 	fsUtilCmdRoot         = "--root"
 	fsUtilCmdLocalStore   = "--local-store-path"
 	fsUtilCmdServerAddr   = "--server-address"
+	fsUtilCmdConnLimit    = "--connection-limit"
 	fsUtilCmdGlobWildCard = "**"
 
 	emptyID = bazel.SnapshotIDPrefix + "-" + bazel.EmptySha + "-0"
 
 	// limit amount of CAS addresses resolved in BzFiler calls to fs_util to this
 	// value, in order to limit number of connections
 	maxResolveToFSUtil = 5
+
+	// connection-limit fs_util flag setting
+	connLimit = 3
```

Comment:
why 3?

Reply:
This is the default (but setting explicit). We don't have a better reason to try setting higher or lower at this point